src/handle-checkout-mutation.js

```diff
@@ -1,8 +1,14 @@
 export default function handleCheckoutMutation(mutationRootKey, client) {
-  return function({data, model}) {
+  return function({data, errors, model}) {
     const rootData = data[mutationRootKey];
     const rootModel = model[mutationRootKey];
-    const userErrors = rootData.userErrors;
+    let userErrors = [];
+
+    if (errors != null && errors.length) {
```

Comment:
don't use coercive equals, but also `errors && errors.length` is sufficient. We don't care if it's `null` or `undefined` or `0`. we just care taht it's falsey